Gemfile.lock

```diff
@@ -1,203 +0,0 @@
-GEM
```

Comment:
I can't remember why we had this in the project to begin with.  Seems like we ended up having to remove it after cloning in CI anyway.  Maybe it was to facilitate the JRuby spec tests in the project but I seem to be able to run those successfully without this in place.  No objections, just curious.